stdlib/public/Concurrency/Task.swift

```diff
@@ -186,6 +186,24 @@ extension Task: Equatable {
   }
 }
 
+@available(SwiftStdlib 5.9, *)
+extension Task where Failure == Never {
```

Comment:
With the appropriate `throws` and `rethrows` annotations, this might just work with throwing functions too? I haven't tested that yet though.

Reply:
Yes